src/types.jl

```diff
@@ -268,6 +268,16 @@ struct File{T<:SUPPORTED_SAMPLE_TYPES,I<:IO}
     signals::Vector{Union{Signal{T},AnnotationsSignal}}
 end
 
+# allow arbitrary collections (AbstractVector or Tuple)
```

Comment:
Could be worth a mention in the documentation somewhere, be it in the existing `File` docstring or in a docstring for this method, that one can pass arbitrary collections of `Signal`s and/or `AnnotationsSignal`s and a copy will be made.

Reply:
[34bc677](https://github.com/beacon-biosignals/EDF.jl/pull/96/commits/34bc67795058aee89944d26a23e1e7d9bfd2e5be)